backend/src/main/java/eu/clarin/switchboard/resources/ToolsResource.java

```diff
@@ -36,14 +34,40 @@ public ToolsResource(ToolRegistry toolRegistry, ToolConfig toolConfig) {
     @Produces(MediaType.APPLICATION_JSON + ";charset=utf-8")
     public Response getTools(@QueryParam("mediatype") String mediatype,
                              @QueryParam("language") String language,
-                             @QueryParam("onlyProductionTools") String onlyProductionTools) {
+                             @QueryParam("onlyProductionTools") String onlyProductionTools,
+                             @Context UriInfo uriInfo) {
+        ProfileMatcher.Builder profileMatcher = ProfileMatcher.builder();
```

Comment:
maybe rename this variable to `profileMatcherBuilder` (or `matcherBuilder` if you want to be short)?